internal/backends/qt/qt_widgets/scrollview.rs

```diff
@@ -46,7 +46,7 @@ impl Item for NativeScrollView {
         cpp!(unsafe [] -> qttypes::QMargins as "QMargins" {
             ensure_initialized();
             QStyleOptionSlider option;
-            initQSliderOptions(option, false, true, 0, 0, 1000, 1000);
+            initQSliderOptions(option, false, true, 0, 0, 1000, 1000, false);
```

Comment:
For `material` and `fluent` style it works like expected but on native the `Slider` is still `horizontal`.

### Example code

```slint
component App inherits Window {
    preferred-width: 700px;
    min-height: 500px;
    title: "Slider";

    VerticalLayout {
        padding: 8px;
        Slider {}
        Slider {is-vertical: true; }
    }
}
```

### Result

<img width="701" alt="image" src="https://github.com/slint-ui/slint/assets/6715107/6d48ebe2-5119-483a-a666-f44440221c67">

Reply:
thanks for the review, I fixed this issue by updating `layout_info` method to return symmetric `LayoutInfo` based on the `vertical` flag.

Also, I managed to test similar code as you did, for both windows 11 and ubuntu 22.04, my env info:

- Windows 11
  - Rust v1.69.0
  - Qt v6.5.1
- Ubuntu 22.04
  - Rust v1.71.1
  - Qt v6.2.4

I tested `VerticalLayout` and `HorizontalLayout` separately with different combinations of sliders, something like this:
```
component AppWindow inherits Window {
    min-width: 700px;
    min-height: 500px;
    title: "Slider";

    VerticalLayout {
        padding: 8px;
        //Slider {}
        //Slider {}
        Slider { vertical: true; }
        Slider { vertical: true; }
    }

    //HorizontalLayout {
    //    padding: 8px;
    //    //Slider {}
    //    //Slider {}
    //    Slider { vertical: true; }
    //    Slider { vertical: true; }
    //}
}

```

For Ubunut 22.04, all looks fine, both vertical and horizontal sliders are displayed as expected, see the screenshot:

![ubuntu_test](https://github.com/slint-ui/slint/assets/1013532/02606869-405b-48b7-812a-565ac84abc53)

For Windows 11, when the orientation of slider is **the same as** orientation of layout, somehow the slider handle is stretched. The same behaviour happened to `master` branch also, so I suppose it's not my change caused this issue, perhaps raise a separate ticket to track it. The test screenshot is:

![windows_test](https://github.com/slint-ui/slint/assets/1013532/73cce6f8-3f02-4d1a-aa46-37123f5916e1)

Reply:
@FloVanGH Hi, need your input here, thanks.

Reply:
@darknight thank you very much. I will take a look tomorrow and give you an answer.